static/app/views/settings/organizationMembers/organizationMembersWrapper.tsx

```diff
@@ -41,12 +41,14 @@ class OrganizationMembersWrapper extends AsyncView<Props, State> {
 
   get hasWriteAccess() {
     const {organization} = this.props;
-    if (!organization || !organization.access) {
-      return false;
-    }
```

Comment:
I noticed that the TS types have `organization` and `access` as non-optional:
https://github.com/getsentry/sentry/blob/7d89cea153e73cf28a8b71c8a8abee2799b00eb1/static/app/types/organization.tsx#L15

So if those are correct we won't need to check that they're available